fio.spec

```diff
@@ -86,6 +86,9 @@ rm -rf $RPM_BUILD_ROOT
 %{_usrsrc}/%{name}-%{version}
 
 %changelog
+* Wed Nov 11 2021 Wang Shilong <shilong.wang@intel.com>
```

Comment:
Format of this line should include the _version_-_release_ such as line 92 does.

You also need to bump the value of the `Release:` on line 3, so as not to replace existing packages with the new ones.